src/launcher/actions/appsActions.js

```diff
@@ -37,6 +37,7 @@
 import { join } from 'path';
 import { ipcRenderer, remote } from 'electron';
 import { ErrorDialogActions } from 'pc-nrfconnect-shared';
+import { sendLauncherUserData, EventAction } from './settingsActions'; // eslint-disable-line import/no-cycle
```

Comment:
Could this import cycle be resolved by moving the new actions to a new file?

Reply:
Resolved by create a new action file.